shared/libpng-util/Makefile

```diff
@@ -1,3 +1,5 @@
+include ../../Config.mk
```

Comment:
you shouldnt need to explicity include this here as every root makefile should include config.mk before this file is ever included

Reply:
I actually already tried installing ENIGMA without including Config.mk in this Makefile and it errored with a reference to a missing "png.h" header. I need to include that so it can look for the headers and libs in the right place.

Reply:
remove it and pastebin the build log

Reply:
Will do

Reply:
https://pastebin.com/rmrENVTw

Reply:
@JoshDreamland do you really care if I include Config.mk in this file? If so, I at least need to add to this Makefile the content from Config.mk that actually allows the headers and libs to be found. Are you ok with me doing either one of these things? BSD installs libs and includes to a different folder than Linux so I need the correct search paths.